install/kubernetes/templates/_container_tetragon.tpl

```diff
@@ -91,5 +91,7 @@
     - mountPath: /etc/tetragon/operator.conf.d/
       name: tetragon-operator-config
       readOnly: true
+  resources:
+    {{- toYaml .Values.tetragonOperator.resources | nindent 4 }}
```

Comment:
I only slightly prefer this approach since it means nothing will be in the template if .Values.tetragon.resources is undefined. But that's a nit.
```
{{- with .Values.tetragon.resources }}
  resources:
    {{- toYaml . | nindent 4 }}
{{- end }}
```